src/modules/feature-modules/admin/pages/users/user-info.component.ts

```diff
@@ -69,94 +52,35 @@ export class PageUserInfoComponent extends CoreComponent implements OnInit {
-          });
-
-          this.unitLength = organisations.filter(org => (response.userOrganisations[0].id === org.id))[0].organisationUnits.length;
-        }
-
-        if (response.type === UserRoleEnum.ACCESSOR || response.type === UserRoleEnum.QUALIFYING_ACCESSOR) {
-          if (response.userOrganisations.length > 0) {
-            this.sections.userInfo = [
-              { label: 'Name', value: response.displayName },
-              { label: 'Type', value: 'Authorised person' },
-              { label: 'Role', value: this.stores.authentication.getRoleDescription(response.userOrganisations[0].role) },
-              { label: 'Email address', value: response.email },
-              { label: 'Phone number', value: response.phone ? response.phone : ' NA' },
-              { label: 'Account status', value: !response.lockedAt ? 'Active' : 'Locked' }
-            ];
-          }
-          this.sections.organisation = response.userOrganisations;
-        }
-        else if ([UserRoleEnum.ASSESSMENT, UserRoleEnum.INNOVATOR].includes(response.type)) {
-          this.sections.userInfo = [
-            { label: 'Name', value: response.displayName },
-            { label: 'Type', value: this.stores.authentication.getRoleDescription(response.type) },
-            { label: 'Email address', value: response.email },
-            { label: 'Phone number', value: response.phone ? response.phone : ' NA' },
-            { label: 'Account status', value: !response.lockedAt ? 'Active' : 'Locked' }
-          ];
-          if (response.type === UserRoleEnum.INNOVATOR) {
-            this.sections.innovations = response.innovations.map(x => x.name);
-            if (response.userOrganisations.length > 0 && !response.userOrganisations[0].isShadow) {
-              this.sections.userInfo = [...this.sections.userInfo,
-              { label: 'Company name', value: response.userOrganisations[0].name },
-              { label: 'Company size', value: response.userOrganisations[0].size }
-              ];
+    this.usersService.getUserInfo(this.activatedRoute.snapshot.params.userId).subscribe({
+      next: (response) => {
+        this.user = {
+          ...response,
+          rolesDescription: response.roles.map(r => {
+            let roleDescription = this.stores.authentication.getRoleDescription(r.role);
+            if (r.role === UserRoleEnum.ASSESSMENT && r.displayTeam) {
+              roleDescription = `${roleDescription} (${r.displayTeam})`;
+            }
+            if (r.role === UserRoleEnum.QUALIFYING_ACCESSOR || r.role === UserRoleEnum.ACCESSOR) {
+              roleDescription = `${roleDescription} (${r.organisationUnit?.name})`;
+            }
+
+            if(r.role === UserRoleEnum.ADMIN && r.isActive) {
+              this.userIsAdmin = true;
+            }
+            if(r.isActive) {
+              this.userHasActiveRoles = true;
+            }
+            if(!r.isActive) {
+              this.userHasInactiveRoles = true;
```

Comment:
On flags I prefer to make it one liner (see if you prefer it as-well):

```
this.userIsAdmin = r.role === UserRoleEnum.ADMIN && r.isActive;
this.userHasActiveRoles = r.isActive;
```

I think you just need one of the flags: `userHasActiveRoles` or `userHasInactiveRoles`. Since one is the opposite of the other.

Reply:
Here I have used both flags because, if a user has active and inactive roles I can show "Active roles" and "Inactive roles" sections. Does that make sense?
![image](https://github.com/nhsengland/innovation-service-transactional-frontend/assets/124596230/9d1be780-6cb4-43db-a8c4-334a2e85c337)

Reply:
I can do somenthing like:        
    
```
this.userIsAdminOrInnovator = r.role === UserRoleEnum.ADMIN || r.role === UserRoleEnum.INNOVATOR;
this.userHasActiveRoles = r.isActive ? true : false;
this.userHasInactiveRoles = !r.isActive ? false : true;
```